src/styles/components/_checkbox.scss

```diff
@@ -0,0 +1,30 @@
+@import '../settings/colors';
+
+.w-checkbox__label {
+  cursor: pointer;
+  display: inline-flex;
+  position: relative;
+}
+
+.w-checkbox__input {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background: url('/images/icons/check_box_outline_blank.svg');
+  border: 2px solid $WHITE;
+  border-radius: 4px;
+  cursor: pointer;
+  height: 28px;
+  margin: 0px 30px 0px 0px;
+  outline: none;
+  transition-duration: .2s;
+  width: 28px;
+}
+
+.w-checkbox__input:checked {
+  background: url('/images/icons/check_box_tertiary.svg');
```

Comment:
One thing I noticed is that this image requires a network request so you see a flash of unstyled content the first time you click the checkbox and the svg loads. Might be better to base64 encode these and include as a data uri. @samthor wdyt?